src/components/InputSection.jsx

```diff
@@ -34,6 +33,10 @@ function InputSection({
                 Get recommendation
               </button>
             </div>
+
+            <div className="md:grow text-sm md:text-base flex flex-col items-center space-y-1 text-center text-primary-dark w-1/2 h-full">
```

Comment:
Do the same thing here, change the `w-1/2` to `lg:w-1/12` This makes larger screens display both of these at 50% width, but screens smaller than that will ignore it and inherit the properties from the parent div above that sets the display to `block`, and width to 100% with `w-full`